Dangerfile

```diff
@@ -1,3 +1,41 @@
 # frozen_string_literal: true
 
-danger.import_dangerfile(gem: 'ruby-grape-danger')
+# Inline checks from ruby-grape-danger (avoids plugins requiring GitHub API token)
```

Comment:
All this, ofc, should be moved to the https://github.com/ruby-grape/danger/blob/master/Dangerfile